spec/controllers/users_controller_spec.rb

```diff
@@ -3,8 +3,9 @@
 RSpec.describe UsersController, type: :controller do
 
  describe "not signed in" do
-     let(:factory_user)  { create(:user) }
-     let(:factory_item)  { create(:item, user: factory_user) }
+     let(:factory_user)         { create(:user) }
+     let(:factory_item)         { create(:item, user: factory_user) }
+     let(:seven_days_old_item)  { Item.create!(name: "old item", created_at: 8.days.ago, user: factory_user)}
```

Comment:
@meghana-singh per my comment on the UserController, we do not need to add any tests to the UserController since all of the destroy code should stay in the rake task. Please remove!